sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/PatchTest.java

```diff
@@ -19,8 +19,7 @@
 import java.util.Objects;
 import java.util.UUID;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.fail;
+import static org.assertj.core.api.Assertions.*;
```

Comment:
We don't use `*` imports, please change this back to individual imports.

Reply:
Fixed.